### ISSUE TITLE
DOSDP question: Can you suppress/override synonyms in a pattern?

### ISSUE THREAD

**Issue by LCCarmody** [monarch-initiative/MAxO#154]
Was trying to set up a pattern of 'Medical examination' or 'Medical specialty examination'. This would apply for 'Medical (specialty) consultation', too.

I can't find an ontology that has the various medical specialties (OMIT and NCIT have a couple).  

So, I thought about building something using UBERON, since many of the examination types are by system.  

I.e. Eye examination, dental examination, neurologic examination, etc.

And also, fetal examination, obstetric examination

Going about it this way is probably not ideal, but my question is, can you suppress synonyms being built using DOSDP?  I think UBERON will not be sufficient for this task, but if using '%s examination', DOSDP would spit out 'late embryonic stage examination' for fetal examination--which clearly would not be appropriate.

I'll start a second ticket about developing a specialty branch.

**Comment by pnrobinson** [comment]
There are lots of specialties that would not fit -- ENT, internal medicine (=many organs), pediatrics, ob/gyn. What about just defining our own classes for this if no other ontology offers them? Reasoning might not work well unless we have good logical definitions for these common specialities.

**Comment by LCCarmody** [comment]
Agree. That's why I think there should be a specialty branch 'somewhere' to fit this issue.  It is an issue for some of the other imaging techniques that would be 'by location'.  WHich is why I posed the question.  I'll see if I can find an example from a better use case.

**Comment by matentzn** [comment]
Agree with @pnrobinson here but regardless of that, yes, you can overwrite synonyms selectively. Please put on meeting minutes for next time and we will go through it!

**Comment by LCCarmody** [comment]
example: 'Ultrasonography_by_location'
'uterine ultrasonography' using the UBERON:0009853 (body of uterus).  So, synonyms using 'ultrasound of %s' create 'ultrasound of body of uterus'.  Honestly, I let it go, but it isn't exactly a term that should be in there.

**Comment by dosumis** [comment]
I think we can give you a way to over-ride all generated synonyms with the same scope (e.g. all exact).

**Comment by LCCarmody** [comment]
I think that would be sufficient for my needs.

**Comment by LCCarmody** [comment]
@matentzn Not sure why this slipped off the radar--given the timing, I'd guess the pandemic might be the reason. But--can we revisit this?

**Comment by matentzn** [comment]
Can you share in dosdp channel on slack?
